js/contact.js

```diff
@@ -0,0 +1,20 @@
+import siteContents from './siteContent.js'
+
+export default function(){
+    let contact = document.querySelector('.contact')
+    let content=siteContents.contact
+    contact.querySelector('h4').textContent=siteContents.contact["contact-h4"]
+    contact.querySelectorAll('p').forEach((item,index)=>{
+        switch(index){
+            case 0:
+                item.innerHTML=content.address
+                break;

```

Comment:
This whole page is very very slick. Everything from the switch cases down to exporting the result of this Lambda Function, your years of experience are shining through here.